test/chi/js/dropdown.pug

```diff
@@ -0,0 +1,258 @@
+---
+title: Dropdown
+---
+script(src='/js/chi.js')
+
+h3 Base
+p  Trigger and dropdown enclosed in the same div.
+.chi-dropdown
+  button#dropdown-1.chi-button.chi-dropdown__trigger Dropdown component
+  .chi-dropdown__menu
+    a.chi-dropdown__menu-item(href='#') Item 1
+    a.chi-dropdown__menu-item(href='#') Item 2
+    a.chi-dropdown__menu-item(href='#') Item 3
+script.
+  chi.dropdown(document.getElementById('dropdown-1'));
+
+h3 Data-target
+p  Dropdown referenced by a data-target attribute in the activator.
+button#dropdown-2.chi-button.chi-dropdown__trigger(data-target='#dropdown-2-menu') Dropdown
+p.-text There is no need for proximity between elements.
+#dropdown-2-menu.chi-dropdown__menu
+  a.chi-dropdown__menu-item(href='#') Item 1
+  a.chi-dropdown__menu-item(href='#') Item 2
+  a.chi-dropdown__menu-item(href='#') Item 3
+  a.chi-dropdown__menu-item(href='#') Item 4
+script.
+  chi.dropdown(document.getElementById('dropdown-2'));
+
+h3 Hover
+p  Dropdown on hover is only supported when trigger and dropdown elements are enclosed in the same div. Use the chi-dropdown__hover class to achieve this.
+.chi-dropdown.chi-dropdown__hover
+  button.chi-button.chi-dropdown__trigger Dropdown component
+  .chi-dropdown__menu
+    a.chi-dropdown__menu-item(href='#') Item 1
+    a.chi-dropdown__menu-item(href='#') Item 2
+    a.chi-dropdown__menu-item(href='#') Item 3
+
+h3 Animate
+p  Triggers can be animated by applying the -animate class.
+.chi-dropdown
+  button#dropdown-animate.chi-button.chi-dropdown__trigger.-animate Animated dropdown trigger
+  .chi-dropdown__menu
+    a.chi-dropdown__menu-item(href='#') Item 1
+    a.chi-dropdown__menu-item(href='#') Item 2
+    a.chi-dropdown__menu-item(href='#') Item 3
+script.
+  chi.dropdown(document.getElementById('dropdown-animate'));
+
+h3 Position
+.example.-mb--3
+  .-p--3.-px-xl--10
+    .chi-grid.-no-gutter.-px-xl--3
+      .chi-col.-w--12.-w-xl--10.-o-xl--1.-d--flex.-justify-content-sm--around.-flex--wrap
+        .-p--1
+          .chi-dropdown
+            button#dropdown-top-start.chi-button.chi-dropdown__trigger.-has-active(data-position='top-start') Top start
+            .chi-dropdown__menu(style='width: 15rem; position: absolute; will-change: transform; right: initial; top: 0px; left: initial; transform: translate3d(0px, -112px, 0px);', x-placement='top-start')
```

Comment:
I believe this has been copied from the generated HTML. No custom styling should be present since Chi Dropdown is supposed to generate styling on fly.

Reply:
Let's remove them from the following test cases as well.

Reply:
Removed custom styling